### PR TITLE
Green button revamp

### PR DESCRIPTION
**Pull request by the-j0k3r**
@silverwind here is a slightly tweaked version which uses more of already existing colors rather than adding new ones.

### CODE REVIEW

**Comment by silverwind** [comment]
Please limit changes to just `github-dark.css`

**Comment by the-j0k3r** [comment]
Was just for testing - removed

**Comment by silverwind** [comment]
Hmm, I prefer your previous (more vibrant) colors more. Maybe use those and possibly adjust existing colors to match if they are reasonably close?

**Comment by the-j0k3r** [comment]
done

**Comment by the-j0k3r** [comment]
Also we should change   /* auto-generated rule for "background-color: #2cbe4e" */ for `#337733` `#373` and it will match the new buttons

**Comment by silverwind** [comment]
Yeah let's update the generator here as well.

**Comment by the-j0k3r** [comment]
Actually a few more `#061` need to be `#373`

 /* auto-generated rule for "background-color: #28a745 !important" */

generic

IDK if there are other places

**Comment by the-j0k3r** [comment]
thats should take care of the 3

https://github.com/StylishThemes/GitHub-Dark/pull/1069/commits/125f9a73a6cd7e97f77b3120857dd51ddedd2f0a

**Comment by silverwind** [comment]
Used this for a bit. May be just me but I'd go one tiny shade darker on the color.

**Comment by the-j0k3r** [comment]
That was what I had initially.

**Comment by the-j0k3r** [comment]
what about instead of `#337733` we go for `#117700`

**Comment by silverwind** [comment]
Hmm yes, `#170` could work too I think.

**Comment by the-j0k3r** [comment]
OK pushed changes =) I really liky :drooling_face:

File: github-dark.css [hunk 1]
```diff
@@ -9105,30 +9105,35 @@
...
   }
   /* Green hovered */
-  .noproblems .pageheader_wrap, a.new-repo:hover {
-    background: #373 !important;
-  }
   .btn-primary:not(:disabled):not(.disabled):not(.compare-button):not(.btn-outline):hover,
   .button.primary:not(:disabled):hover, .btn-theme-green:hover,
   .btn-primary-mktg:hover {
-    background: linear-gradient(#508055, #407045) !important;
+    background: linear-gradient(180deg, #170 5%, #170 100%) !important;
```
Comment (silverwind) [inline github-dark.css]:
```suggestion
    background: linear-gradient(180deg, #280 5%, #170 100%) !important;
```

Hover should not remove gradient.

**Comment by silverwind** [comment]
Hmm I'm not 100% sure yet. It does look okay on one of my screens, but too flashy on another 😱

**Comment by the-j0k3r** [comment]
Yea I have same issue with my laptop screen, the colors are not same as in my desktop screen

Desktop screen represents the colors faithfully Its calibrated IPS screen, while laptop screen is not anywhere close, I its a TN screen.

Overall these offer the best balance.

File: github-dark.css (hunk 1 above)
Comment (the-j0k3r) [reply github-dark.css]: Thats far too much, even `#180` is already too much

Comment (the-j0k3r) [reply github-dark.css]: How about
```suggestion
    background: linear-gradient(180deg, #180 1%, #170 70%) !important;
```

Comment (the-j0k3r) [reply github-dark.css]: lol whats with the orange

Comment (the-j0k3r) [reply github-dark.css]: <s>lol whats with the orange<\s>

Never mind GH is misbehaving.

**Comment by the-j0k3r** [comment]
Ive just also noticed we dont have any

```
.btn-primary.selected, .btn-primary:active, .btn-primary[aria-selected="true"], [open] > .btn-primary {
	background-color: #279f43;
	background-image: none;
	border-color: rgba(27,31,35,.5);
	box-shadow: inset 0 .15em .3em rgba(27,31,35,.15);
}
```

generic

**Comment by silverwind** [comment]
I'm sorry but can you restore your previous iteration once more? (With the gradient fix if possible).

**Comment by the-j0k3r** [comment]
what `#373`?

**Comment by the-j0k3r** [comment]
done, though this isnt better than before, hover gradient is either too bright or too dark on top or bottom or both, also, I think its best to use the temp usercss to test colors, this back and forth is consuming too much time.

**Comment by silverwind** [comment]
> this back and forth is consuming too much time.

you better get used to the branching model 😆

**Comment by the-j0k3r** [comment]
> you better get used to the branching model laughing

It was what was done here =) I meant making changes and not adding silly commits for each change

**Comment by xt0rted** [comment]
The primary buttons seem too bright now. They stand out from the other greens more than the old colors did and they're the brightest thing on the page. The old colors seemed better here.

**Comment by silverwind** [comment]
Don't think you'll get used to them? I think the old ones looked a bit "dull".

**Comment by xt0rted** [comment]
They just feel too vibrant compared to the rest of the color palette.

**Comment by silverwind** [comment]
Maybe we can make them a bit duller, but not as dull as before 😉

**Comment by the-j0k3r** [comment]
> Maybe we can make them a bit duller, but not as dull as before wink

ugh lol

**Comment by silverwind** [comment]
I don't think many users would want to configure button colors 😆

**Comment by the-j0k3r** [comment]
I dont do styles for others, though they a re welcome to decide to use them or not.

**Comment by silverwind** [comment]
Will likely do a slight revision of the color to desaturate it a bit.

**Pull request merged**